swiftpkg/internal/swiftpkg_build_files.bzl

```diff
@@ -435,7 +438,7 @@ def _clang_target_build_file(repository_ctx, pkg_ctx, target):
         noop_modulemap = clang_src_info.modulemap_path != None
         modulemap_attrs = {
             "deps": bzl_selects.to_starlark(modulemap_deps),
-            "hdrs": clang_src_info.hdrs,
+            "hdrs": [] if noop_modulemap else clang_src_info.hdrs,
```

Comment:
Not needed but noticed we were adding all the hdrs here even in the noop case which seems wasteful.